src/command_modules/azure-cli-cosmosdb/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -186,6 +188,11 @@ def cli_cosmosdb_update(client,
 
     if enable_multiple_write_locations is None:
         enable_multiple_write_locations = existing.enable_multiple_write_locations
+    elif enable_multiple_write_locations != existing.enable_multiple_write_locations and \
+            enable_multiple_write_locations:
+        raise CLIError("Cannot convert account from single master to multi master")
+    elif enable_multiple_write_locations != existing.enable_multiple_write_locations:
+        print("This operation will take 24 hours to complete")
```

Comment:
This has to be printed to stderr. You can either use logger.warning or just add the necessary kwargs to print to stderr.